src/index.js

```diff
@@ -100,6 +103,8 @@ class DockAPI {
     // Initialize price feed rpc
     rpc = Object.assign(rpc, PriceFeedRpcDefs);
 
+    rpc = Object.assign(rpc, CoreModsRpcDefs);
```

Comment:
we can do:
```
rpc = Object.assign(Object.assign(rpc, PriceFeedRpcDefs), CoreModsRpcDefs);
```

Reply:
Is there an advantage to doing this. Because i like them on separate lines for readability. Added a comment btw

Reply:
personal preference, can keep it either way